corc-core/src/main/java/com/hotels/corc/mapred/CorcInputFormat.java

```diff
@@ -1,5 +1,10 @@
 /**
- * Copyright (C) 2015-2016 Expedia Inc.
+ * Copyright (C) 2015-2019 Expedia Inc and the original spring-integration contributors.
```

Comment:
It's not `spring-integration` we took the code from in this case, it's the `Hive project`.